prow/config.yaml

```diff
@@ -41,6 +41,10 @@ plank:
     config:
       default_service_account_name: "prowjob-default-sa" # Use workload identity
       gcs_credentials_secret: ""                         # rather than service account key secret
+  - cluster: prow-arm-private
+    config:
+      default_service_account_name: "prowjob-default-sa" # Use workload identity
+      gcs_credentials_secret: ""                         # rather than service account key secret
```

Comment:
should be like the istio-private one

```
      default_service_account_name: "prowjob-private-sa"
      gcs_configuration:
        bucket: "istio-private-build"
        path_strategy: "explicit"
      ssh_key_secrets:
      - ssh-key-secret
```

Reply:
weird, need to look into the docs. Regardless prowjob-default-sa seems wrong , seems like we have a private SA?

Reply:
you can match org or repo (confusing key, it's actually matching the org). https://github.com/kubernetes/test-infra/blob/c66af3fdd561/prow/config/config.go#L721

so we. probably want to remove this entirely